python/cuopt_server/cuopt_server/utils/job_queue.py

```diff
@@ -269,8 +269,12 @@ def get_solution_for_id(id, delete=True):
                 s = saved_results[id]
                 if delete:
                     del saved_results[id]
-                return s.wait() + (mime_type,)
+                if timestamps:
+                    return s.wait(timeout=10) + (mime_type, s.get_timestamps())
```

Comment:
This doesn't seem to be having any effect and I'm not sure why. The goal was to just hardcode in a wait time so that we don't have to poll at such a low interval.  Will investigate more ....